__tests__/api-categories.test.js

```diff
@@ -1,20 +1,52 @@
 process.env.NODE_ENV = "test";
 const request = require("supertest");
 const  app  = require("../app");
-const db = require("../db/data/test-data");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data")
+
+
+beforeEach(() => {
+	return seed(testData);
+});
 
 afterAll(() => {
 	if (db.end) db.end();
 });
 
+//do beforeaach on onbe line 
+
+
 describe("1. GET /api/categories", () => {
-      test("GET /api/categories returns an object", () => {
+      test("200: GET /api/categories returns an array", () => {
 				return request(app)
 					.get("/api/categories")
 					.expect(200)
 					.then((res) => {
-						const { body } = res;
-						expect(typeof body).toBe("object");
+                        const { body } = res;
+                        console.log(body.categories, "body")
+						expect(Array.isArray(body.categories)).toBe(true);
+                    });
+          
+      });
+    test("200: returns all of the available treasures", () => {
```

Comment:
I appreciate you were probably using this for reference, but we have no place for treasures here!

Reply:
Ah, I see what you've done - you could probably mesh the first two tests together, to be something like "200 and responds with an array of categories. Then include the stuff from lines 38-42 in your test above. But still scrub out the 'treasure' bits please.